libraries/bxdf/gltf_pbr.mtlx

```diff
@@ -25,7 +25,7 @@
     <input name="emissive" type="color3" value="0, 0, 0" uimin="0, 0, 0" uimax="1, 1, 1" uiname="Emissive" uifolder="Emission" />
     <input name="emissive_strength" uniform="true" type="float" value="1" uimin="0" uiname="Emissive Strength" uifolder="Emission" />
     <input name="thickness" uniform="false" type="float" value="0" uimin="0" uiname="Thickness" uifolder="Volume" />
-    <input name="attenuation_distance" uniform="true" type="float" uimin="0" uiname="Attenuation Distance" uifolder="Volume" />
+    <input name="attenuation_distance" uniform="true" type="float" value="1e+36" uimin="0" uiname="Attenuation Distance" uifolder="Volume" />
```

Comment:
Perhaps this collection of data library fixes should become its own pull request?  Specifically for this one, I believe we should bring Khronos into the process of validating it.

Reply:
Yes. I can move this out.